conf/portal.application.conf

```diff
@@ -205,6 +205,10 @@ reporting {
   }
 }
 
+alerting {
+  execution.defaultInterval = 30s
+}
+
```

Comment:
purely out of curiosity, do we have a mechanism to detect the case where:

we have 40 seconds of work we're trying to schedule every 30 seconds

?

Reply:
We don't have any proactive mechanism, no. The current use case is reporting and runs significantly less frequently than alerting, at most daily, and so it would likely never run into this.

This is an interesting question though and we might want to force a timeout on jobs to make sure this doesn't happen. I definitely intend to instrument any scheduling lag (e.g. ∆t_lag = scheduled - now < 0) if it isn't already present.